external/HavoqGT/README.md

```diff
@@ -1,26 +1,31 @@
 
 # Required to Build HavoqGT with UMap
 
+We assume that the following items are already available (installed) on the system:
 - GCC 8.1 or more.
 - MPI
 - CMake 2.6 or more.
-- [Boost C++ Libraries](https://www.boost.org/) 1.64 or more.
-- [Metall](https://github.com/LLNL/metall)
+
+Since HavoqGT is not a read-only application,
+the write-protect support for userfaultfd() must be supported on the system.
 
```

Comment:
Maybe phrase it as:
HavoqGT creates, analyzes, modifies, and saves graphs. To support this usage, the write-protect feature of userfaultfd() must be available on the system's Linux kernel.